### DISCUSSION TITLE
edit nginx-proxy config

### DISCUSSION THREAD

**Discussion by nguyenphong2001** [EasyEngine/easyengine#1590] [General]
Hi guys. 
I want to change something in the config of nginx-proxy. It is located here "/opt/easyengine/services/nginx-proxy/conf.d/default.conf". But every time I reload or restart nginx-proxy, everything is automatically reset to the original setting. All my changes are gone. So how can I edit them? Thanks

**Comment by cerbenus** [comment]
Check this out:

[https://easyengine.io/handbook/internal/nginx-proxy/](url)

Don't believe we should be messing with this file since it's used as a reversed proxy. My understanding is that Nginx is used as a load balancer here.

Use this file instead:

`/opt/easyengine/sites/mysite.com/config/nginx/custom/user.conf`
